### DISCUSSION TITLE
`getHomeFeed` - Recommendations are not based on my account.

### DISCUSSION THREAD

**Discussion by SupertigerDev** [LuanRT/YouTube.js#190] [Q&A]
I tried this:

```js
const home = await yt.getHomeFeed();

home.videos.forEach(v =>  {
  console.log(v.title.text)
})
```
The video titles that are returned are not recommendations that are based on my account. They seem like random videos. Is there anyway to get recommended videos that are based on my account? I logged in using `cacheCredentials`

**Comment by LuanRT** [comment]
Hey, could you verify that you are in fact signed in? I just checked and got relevant recommendations.

Try any method that requires sign-in, example:
```ts
const info = await yt.account.getInfo();
console.log(info);
```

If they throw an error then there's probably something wrong with the cached credentials.

**Reply by SupertigerDev** [reply]
Sorry, forgot to mention that I tried that. It returns my YouTube channel name and other stuff yes.
